app/controllers/api/v1/card_decks_controller.rb

```diff
@@ -0,0 +1,47 @@
+class Api::V1::CardDecksController < ApplicationController
+  
+  # GET /api/v1/card_decks
+  def index
+    card_decks = CardDeck.all
+    card_decks_json = ActiveModel::Serializer::CollectionSerializer.new(card_decks, each_serializer: CardSerializer)
+    render json: { card_decks: card_decks_json }, status: :ok
+  end
+  
+  # POST /api/v1/decks/:deck_id/cards/:card_id/card_decks
+  def create
+    deck = Deck.find(params[:deck_id])
+    deck.user = current_user
+    card = Card.find(params[:card_id])
+    card.user = current_user
+    
+    card_deck = CardDeck.new(card_deck_params)
+    card_deck.card_id = card.id
+    card_deck.deck_id = deck.id
+    
+    if card_deck.save
```

Comment:
Where is your check if this `CardDeck` already exist? If client is sending this to you 5 times, you should not throw an error. This API endpoint should be idempotent.